cmd/minikube/cmd/kubectl.go

```diff
@@ -84,6 +104,33 @@ var kubectlCmd = &cobra.Command{
 	},
 }
 
+func installKubectl(binary, version, path, prefix string) {
+	bindir := filepath.Join(prefix, "bin")
+	installpath := filepath.Join(bindir, binary)
+	glog.Infof("Installing %s (%s)", installpath, version)
+
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		exit.WithError("Failed to read kubectl", err)
+	}
+	err = os.MkdirAll(bindir, 0755)
+	if err != nil {
+		exit.WithError("Failed to create directory", err)
+	}
+	err = ioutil.WriteFile(installpath, data, 0755)
+	if err != nil {
+		if os.IsPermission(err) {
+			// Permission denied (common error) is *not* a crash...
+			console.Fatal("%s: %v", "Failed to write kubectl", err)
+			os.Exit(exit.Software)
+		}
+		exit.WithError("Failed to write kubectl", err)
+	}
+}
+
 func init() {
+	kubectlCmd.Flags().BoolVar(&kubectlInstallMode, "install", false, "Install kubectl and exit")
+	kubectlCmd.Flags().StringVar(&kubectlInstallPrefix, "prefix", defaultPrefix, "Installation prefix")
+	kubectlCmd.Flags().BoolVar(&kubectlPathMode, "path", false, "Display kubectl path instead of running it")
```

Comment:
without reading the help command I would consider that `path` would be the place to install, pretty much like `prefix`.
what do you think about `--which` cos it does the same thing as `which kubectl`, right? probably wouldn't make the same sense for a windows user.

Reply:
Yeah, that is one idea. I copied `--url` (from dashboard), and then it became `--path`.

Reply:
Think `--which` can be misleading, since it is normally associated with PATH (which it is not in). Will go for something like `--print`, and hope that there is no confusion.